### DISCUSSION TITLE
Not able to find 5G SA in iphone 12

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#269] [General]
hi @herlesupreeth i am using iphone 12 (IOS version:17.2.1) i am able to attach volte and voice call is established successfully. 

now i try to attach 5G SA with plmn 001/01  .when i am doing manual network search i am not able to find the network name also .

i tried with apn default, ims and internet also but no luck.  please help me

**Comment by herlesupreeth** [comment]
Please read the post here - https://github.com/herlesupreeth/docker_open5gs/discussions/248#discussioncomment-7392233

and ensure you have enabled VoLTE option in the phone

I tested only with PLMN 001 01 and it worked

**Reply by Sivanesh1992** [reply]
hi @herlesupreeth   i directly programmed opencell simcard with mcc(001) and mnc( 01) and volte option also enabled in iphone12 but i am not able to see network name .  i gone the command 248 .  

i have one query.
 we need to give apn as internet or default  because we update apn internet and ims in webgui .

**Reply by herlesupreeth** [reply]
>  i gone the command 248 .

I didnt understand what you mean here. 

> i directly programmed opencell simcard with mcc(001) and mnc( 01)

I would suggest programming the SIM to 999 70 PLMN + program EF.SUCI_Calc_Info along with enabling UST service 124 and disable 125. Then, plug in the SIM into iPhone and enable SA mode in iPhone settings and attach to the gNB running with PLMN 999 70 (dont forget the enabling encryption parts as mentioned in that post both on AMF and gNB). While using 999 70 PLMN phone does not need Voice support. Once the phone attaches and you are able to do data. Then, shutdown the network and re-program the SIM to 001 01 and change PLMN of the network to 001 01 and enable VoLTE. You would have to do a manual network search on the phone. Using a GPSDO (e.g. from Leo Bodnar) would help a lot in quick attach.

**Reply by Sivanesh1992** [reply]
hi @herlesupreeth  happy new year. thankyou let me try and get back to you
